src/backend/controllers/pages.ts

```diff
@@ -62,6 +62,47 @@ class Pages {
     return nullFilteredPages;
   }
 
+  /**
+   * Group given pages with descending order of creation time
+   *
+   * @param {pages[]} pages - pages to group
+   * @returns {Page[]}
+   */
+  public static group(pages: Page[]): Page[] {
+    const result: Page[] = [];
+    const obj:Record<string, Array<Page>> = {};
+
+    pages.sort((a, b) => {
```

Comment:
please, move the handler to a separate method like `sortByTimeDesc`

Reply:
Fixed